protected_test.go

```diff
@@ -25,7 +25,7 @@ func (p *testprotector) Protect(fileDescriptor int) error {
 
 func TestConnectIP(t *testing.T) {
 	p := &testprotector{}
-	pt := New(p.Protect, "8.8.8.8")
+	pt, _ := New(p.Protect, "8.8.8.8:53")
```

Comment:
Sorry for the late reply @oxtoacart. It won't default port to 53. I know it's a bit weird, but it just follows the Go convention and saves a few lines of code by reusing the same `splitHostPort` method.